bigchaindb/core.py

```diff
@@ -25,7 +25,7 @@ class Bigchain(object):
     TX_IN_BACKLOG = 'backlog'
     """return if transaction is in backlog"""
 
-    def __init__(self, connection=None):
+    def __init__(self, tendermint_host=None, tendermint_port=None, connection=None):
```

Comment:
I don't think we need this, the user can use the config file `.bigchaindb` or the environment variables.